performance_manager/lib/s3_utils.py

```diff
@@ -17,7 +17,9 @@ def _get_pyarrow_table(
     """
     internal function to get pyarrow table from parquet file(s)
     """
+
     active_fs = fs.S3FileSystem()
+
```

Comment:
this is all some reformatting. i had put in some local fs stuff for local tests, but we decided monkey patching was a better way to go.